Binary_Tree_Sort_C/main.c

```diff
@@ -0,0 +1,86 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+struct Node
+{
+    int value;
+    struct Node *leftChild;
+    struct Node *rightChild;
+};
+
+struct Node *newNode(int number, struct Node *tree)
+{
+    if (tree == NULL)
+    {
+        tree = malloc(sizeof (struct Node));
+        tree->value = number;
+        tree->leftChild = NULL;
+        tree->rightChild = NULL;
+    }
+    else
+    {
+        if (number < tree->value)
+        {
+            tree->leftChild = newNode(number, tree->leftChild);
+        }
+        if (number > tree->value || number == tree->value)
+        {
+            tree->rightChild = newNode(number, tree->rightChild);
+        }
+    }
+    return tree;
+}
+
+int main()
+{  
+    void SortArray(struct Node *binaryTree, int **p); // function prototypes
```

Comment:
I believe function prototypes should be at the beginning of the file after the includes.